docs/proposals/kubelet-eviction.md

```diff
@@ -76,8 +76,10 @@ The `kubelet` will support the ability to trigger eviction decisions on the foll
 | memory.available | memory.available := node.status.capacity[memory] - node.stats.memory.workingSet |
 | nodefs.available   | nodefs.available := node.stats.fs.available |
 | nodefs.inodesFree | nodefs.inodesFree := node.fs.inodesFree |
+| nodefs.inodesPercentAvailable | nodefs.inodesPercentAvailable := node.fs.inodesFree / node.fs.inodes |
```

Comment:
Would that mean you'd need to make it a `string` or would you use some custom marshaling/unmarshaling?